dali/pipeline/operators/util/randomizer_impl_cpu.h

```diff
@@ -25,7 +30,14 @@ Randomizer<CPUBackend>::Randomizer(int seed, size_t len) {}
 
 template <>
 int Randomizer<CPUBackend>::rand(int idx) {
+#if !defined(__AARCH64_QNX__)
   return lrand48();
```

Comment:
@ptrendx - do you know why we are using lrand48(); and not std::random_device rd;?
Maybe we could get rid of lrand48 after all?

Reply:
Yes, we could. I don't think Randomizer is actually used on the CPU (I think it was only used by Jitter but I disabled it because it was not deterministic there).

Reply:
@JanuszL @ptrendx : Should I remove lrand48 part?